.pre-commit-config.yaml

```diff
@@ -51,25 +54,20 @@ repos:
       - id: black
       # - id: black-jupyter  # Consider adding this
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.9.6
+    rev: v0.11.5
     hooks:
       - id: ruff
         args: [--fix-only, --show-fixes]  # --unsafe-fixes]
   - repo: https://github.com/PyCQA/flake8
-    rev: 7.1.1
+    rev: 7.2.0
     hooks:
       - id: flake8
         args: ["--config=.flake8"] # uses config
         additional_dependencies: &flake8_dependencies
           # These versions need updated manually
-          - flake8==7.1.1
+          - flake8==7.2.0
           - flake8-bugbear==24.12.12
-          - flake8-simplify==0.21.0
-  - repo: https://github.com/asottile/yesqa
-    rev: v1.5.0
-    hooks:
-      - id: yesqa
-        additional_dependencies: *flake8_dependencies
+          - flake8-simplify==0.21.0  # Not maintained, so consider deleting
```

Comment:
fwiw `ruff` has all of the `flake8-simplify` and `flake8-bugbear` checks (among others) and is maintained

Reply:
Thanks for the quick review!

I know this is approximately true and maybe asymptotically true, but `flake8-bugbear` in particular has caught things in the past that `ruff` didn't, and it's well-maintained, so I don't mind keeping it. Happy to delete `flake8-simplify` though.